packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -136,10 +140,20 @@ export class AggregatedAttestationPool {
       attestationGroupByIndex = new Map<CommitteeIndex, MatchingDataAttestationGroup>();
       attestationGroupByIndexByDataHash.set(dataRootHex, attestationGroupByIndex);
     }
-    const committeeIndex = isElectraAttestation(attestation)
-      ? // this attestation is added to pool after validation
-        attestation.committeeBits.getSingleTrueBit()
-      : attestation.data.index;
+
+    let committeeIndex;
+
+    if (isForkPostElectra(this.config.getForkName(slot))) {
+      if (!isElectraAttestation(attestation)) {
+        throw new Error("Attestation should be type electra.Attestation");
```

Comment:
I think we could still keep the slot in case we need to debug this further

```suggestion
        throw Error(`Attestation should be type electra.Attestation for slot ${slot}`);
```